src/app/work-item/work-item-list/work-item-detail/work-item-link/work-item-link.component.html

```diff
@@ -1,132 +1,132 @@
-                                        class="pointer" 
-                                        (click)="selectSearchResult(workItem.id, workItem.attributes['system.title'])"
-                                        [attr.data-wiid]="workItem.id"
-                                        [attr.data-wititle]="workItem.attributes['system.title']">
-                                        <span #searchresult><span class="weight-800">{{workItem.id}} - </span>{{workItem.attributes['system.title']}}</span>
-                                    </li>
-                                </ul>
-                            </div>
-                        </div>
-                        <div class="col-md-3 col-sm-3 col-xs-3">
-                            <button [disabled]="!selectedLinkType || !selectedWorkItemId ? true : false" 
-                              type="button" 
-                              value="Link" 
-                              class="col-md-12 col-sm-12 col-xs-12 btn btn-primary" 
-                              (click)="createLink($event)">
-                              Link
-                            </button>
-                        </div>
+        <div  *ngIf="showLinkCreator" class="link-editor" id="wi-link-editor">
+            <div class="col-md-12 col-sm-12 col-xs-12">
+              <div dropdown class="dropdown detail-status-dropdown col-md-6 col-sm-6 col-xs-6">
+                <button dropdown-open
+                  id="wi-link-type"
+                  type="button"
+                  class="btn dropdown-toggle col-md-12 col-sm-12 col-xs-12 btn-default">
+                  <span *ngIf="!selectedLinkType && (workItem.relationships.baseType.data.id | almValidLinkTypes:linkTypes).length" class='dropdown-text'>
+                    Select a link type
+                  </span>
+                  <span *ngIf="!(workItem.relationships.baseType.data.id | almValidLinkTypes:linkTypes).length" class='dropdown-text'>
+                    No link type available
+                  </span>
+                  <span *ngIf="selectedLinkType" class='dropdown-text'>
+                    {{selectedLinkType.name}}
+                  </span>
+                  <span *ngIf="loggedIn" class="caret pull-right marginT5"></span>
+                </button>
+                <ul *ngIf="(workItem.relationships.baseType.data.id | almValidLinkTypes:linkTypes).length"
+                  class="dropdown-menu dropdown-ul link-dropdown" role="menu">
+                  <li *ngFor="let linkType of selectedTab |
+                    workItemLinkTypeFilterByTypeName:(
+                      workItem.relationships.baseType.data.id |
+                        almValidLinkTypes:linkTypes)">
+                    <a (click)="onSelectRelation(linkType)">{{linkType.name}}</a>
+                  </li>
+                </ul>
+              </div>
+              <span class="col-md-6 col-sm-6 col-xs-6"> to,item</span>
```

Comment:
Not sure about the wording.
Probably not needed at all, as the dropdown already lists `by, to` etc suffixes.